src/blssignature.hpp

```diff
@@ -66,30 +69,24 @@ class BLSSignature {
     // be verified.
     void SetAggregationInfo(const AggregationInfo &newAggregationInfo);
 
-    const uint8_t& operator[](size_t pos) const;
-    const uint8_t* begin() const;
-    const uint8_t* end() const;
-    size_t size() const;
-
     // Serializes ONLY the 96 byte public key. It does not serialize
     // the aggregation info.
     void Serialize(uint8_t* buffer) const;
+    std::vector<uint8_t> Serialize() const;
```

Comment:
If we want to keep both serialize, we should also do it for other classes like ExtendedPubKey, ChainCode, etc